mtl/psmtl2milp.py

```diff
@@ -0,0 +1,172 @@
+'''
+ Explainable Robotics Lab, Lehigh University
+ See license.txt file for license information.
+ @author: Gustavo A. Cardona, Cristian-Ioan Vasile
+'''
+
+import gurobipy as grb
```

Comment:
Add a blank line between the two imports to showcase that one is a third party package (i.e., gurobipy) and the other import is local (from the same package).